data/default_release_images.json

```diff
@@ -55,7 +55,7 @@
       "version": "4.11.0"
     },
     {
-        "openshift_version": "4.11",
+        "openshift_version": "4.11-multi",
```

Comment:
We don't want to have ocp-bump for multi today because it's not supported. There is no upgrade graph for multi yet and I will fix it via https://github.com/openshift-assisted/assisted-installer-deployment/pull/293 as soon as there is a stable solution

>Perhaps we should have a distinct 4.11 copy then?

We had it before. We need to have different value of `openshift_version` for single-arch and multi-arch, otherwise they will be treated as the same by the UI